src/api.ts

```diff
@@ -22,4 +22,4 @@ function get<T>(url: string): Promise<T> {
 
 export const getTodos = () => get<Todo[]>('/todos');
 
-export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+export const getUser = (userId: number) => get<User>(`/users/${userId}`);
```

Comment:
I think it will be more readable)
```suggestion
export const getUserById = (userId: number) => get<User>(`/users/${userId}`);
```

Reply:
> I think it will be more readable)

Not fixed